store/storetest/file_info_store.go

```diff
@@ -276,16 +283,19 @@ func testFileInfoAttachToPost(t *testing.T, ss store.Store) {
 
 		err = ss.FileInfo().AttachToPost(info1.Id, postId, userId)
 		assert.Nil(t, err)
+		info1.PostId = postId
 
 		err = ss.FileInfo().AttachToPost(info2.Id, postId, userId)
 		assert.Nil(t, err)
+		info2.PostId = postId
 
 		data, err := ss.FileInfo().GetForPost(postId, true, false, false)
-		assert.Nil(t, err)
+		require.Nil(t, err)
 
-		assert.Len(t, data, 2)
-		assert.True(t, data[0].Id == info1.Id || data[0].Id == info2.Id)
-		assert.True(t, data[1].Id == info1.Id || data[1].Id == info2.Id)
+		expected := []*model.FileInfo{info1, info2}
+		sort.Sort(byFileInfoId(expected))
```

Comment:
Interesting. I've never seen this function used this way before since I usually just use `sort.Slice`. I wouldn't have thought it still modifies this underlying slice even though it's cast to that type.

Reply:
Heh, I'll admit I've never used `sort.Slice`! Odd that I've never encountered it -- it actually probably makes slightly more sense here, but I won't bother changing unless you think it's worth it.